pkg/volume/csi/csi_block.go

```diff
@@ -56,12 +56,12 @@ func (m *csiBlockMapper) GetGlobalMapPath(spec *volume.Spec) (string, error) {
 }
 
 // GetPodDeviceMapPath returns pod's device file which will be mapped to a volume
-// returns: pods/{podUid}/volumeDevices/kubernetes.io~csi/{volumeID}/dev, {volumeID}
+// returns: pods/{podUid}/volumeDevices/kubernetes.io~csi/{volumeID}/dev, file
 func (m *csiBlockMapper) GetPodDeviceMapPath() (string, string) {
 	path := filepath.Join(m.plugin.host.GetPodVolumeDeviceDir(m.podUID, csiPluginName), m.specName, "dev")
-	specName := m.specName
-	glog.V(4).Infof(log("blockMapper.GetPodDeviceMapPath [path=%s; name=%s]", path, specName))
-	return path, specName
+	fileName := "file"
```

Comment:
Returning the value `"file"` seems extremely specific and is may not be useful for all drivers .  Returning the volumeID may be needed during operations such as tearDown.